mission1/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="판다 마켓" />
+    <meta property="og:description" content="일상의 모든 물건 을 거래해보세요" />
+    <meta property="og:url" content="/" />
+    <link rel="stylesheet" href="style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <div id="wrap">
+      <!-- HEADER -->
+      <div class="header_wrap">
+        <header class="header">
+          <h1 class="logo">
+            <a href="/">판다마켓</a>
+          </h1>
+          <div class="login_box">
+            <a href="login.html" class="login_btn">로그인</a>
+          </div>
+        </header>
+      </div>
+      <!-- // HEADER -->
+
+      <!-- MAIN -->
+      <main>
+        <section class="sec01">
+          <div class="inner">
+            <div class="visual_box">
+              <h2>일상의 모든 물건을 거래해 보세요.</h2>
+              <a href="items.html" class="btn round bg_primary_100 large">구경하러 가기</a>
+            </div>
+          </div>
+        </section>
+        <section class="sec02">
+          <div class="inner">
+            <div class="info_box">
+              <span class="tit">Hot item</span>

```

Comment:
💊 제안
단축어를 사용하기도 하지만 일반적인 경우 단어를 줄이지 않고 사용하는 것이 더 직관적이고 가독성이 좋다고 생각합니다~

```suggestion
              <span class="title">Hot item</span>
```